src/components/POLine/POLineForm.js

```diff
@@ -157,6 +158,29 @@ function POLineForm({
     ));
   };
 
+  // eslint-disable-next-line react/prop-types
+  const getActionMenu = ({ onToggle }) => (
+    Boolean(templateValue?.hiddenFields) && (
+      <MenuSection id="po-line-form-actions">
+        <IfPermission perm="ui-orders.order.showHidden">
+          <Button
+            id="clickable-show-hidden"
+            data-testid="show-hide-btn"
```

Comment:
I'd suggest to rename to `toggle-fields-visibility`

Reply:
Renamed